pydantic_ai_slim/pydantic_ai/_utils.py

```diff
@@ -415,6 +417,20 @@ def merge_json_schema_defs(schemas: list[dict[str, Any]]) -> tuple[list[dict[str
     return rewritten_schemas, all_defs
 
 
+def validate_no_deprecated_kwargs(_deprecated_kwargs: dict[str, Any]) -> None:
```

Comment:
Can we drop `deprecated` from the names and description? If there are remaining items in the dict here, they wouldn't be _deprecated_ kwargs, just unknown kwargs.